test/e2e/pods.go

```diff
@@ -689,7 +689,7 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 15,
+							InitialDelaySeconds: 30,
 							FailureThreshold:    1,
```

Comment:
Alternatively, we could increase the `FailureThreshold`, although I think that's probably more likely to mask legitimate bugs.